project_key/tests/test_common.py

```diff
@@ -9,8 +9,9 @@ class TestCommon(TransactionCase):
     def setUp(self):
         super(TestCommon, self).setUp()
 
-        self.Project = self.env['project.project']
-        self.Task = self.env['project.task']
+        test_ctx = dict(test_project_key=True)
```

Comment:
Don't do it this way, as you are replacing full context dictionary passing a dictionary argument to next lines.